README.md

```diff
@@ -38,10 +38,10 @@ root
 │   ├── <a href="./packages/message-relayer">message-relayer</a>: Tool for automatically relaying L1<>L2 messages in development
 │   └── <a href="./packages/replica-healthcheck">replica-healthcheck</a>: Service for monitoring the health of a replica node
 ├── <a href="./go">go</a>
```

Comment:
If we want to be 100% correct, we should reformat this tree to match the new folder structure

Reply:
This is being reformatted in the larger monorepo PR :+1: